decred/decred/dcr/dcrdata.py

```diff
@@ -421,6 +417,8 @@ def __init__(self, db, params, datapath, skipConnect=False):
         self.txBlockMap = db.child("blocklink")
         self.tipHeight = None
         self.subsidyCache = calc.SubsidyCache(params)
+        self.addrSubscribers = {}
+        self.blockSubscribers = []
```

Comment:
Different receivers for different addresses allows the `DcrdataBlockchain` to be reused by different accounts.